File: pset6/mario/mario.py
from cs50 import get_int


def main():
    while True:
        height = get_int("Height: ")
        if height > 0 and height < 9:
            break
    for row in range(0, height):
        for length in range(0, height):
            if row + length < height - 1:
                print(" ", end="")
            else:
                print("#", end="")
        print()


main()

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Nested for loops to print hashes
    for (int row = 0; row < height; row++)
    {
        for (int column = 0; column < height; column++)
        {
            if (column < height - row - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("  ");
        // For reverse hashes
        for (int col_rev = 0; col_rev < height; col_rev++)
        {
            if (row > col_rev - 1)
            {
                printf("#");
            }
        }
        printf("\n");
    }
}